### PR TITLE
PX-Backup 1.1 RN

### PR DESCRIPTION
**Pull request by andreipope**
Init

### CODE REVIEW

**Comment by andreipope** [comment]
@nathan-px @bhavana-ar This is still a draft. Please see my comments.

File: content/release-notes.md [hunk 1]
```diff
@@ -16,49 +16,49 @@ Sep 23, 2020
...
+| PB-747 | Administrators can now share backup locations with other users without disclosing the credentials or the bucket name by setting a backup location as the default backup location. <!-- TODO: Revisit this -->|
 | PB-768 | Users can now delete a resource without being prompted for the name of that resource. |
+| PB-636 | Improved clarity around the `OrgID` field in the **Add License** view. |
+| PB-645 | In the **Edit Backup Schedule View**,  you can now use a navigation link to go to the **Schedule Policy** view. |
+| PB-710 | Added a tooltip showing whether a backup schedule is being deleted. |
+| PB-509 | When your Keycloak token expires, PX-Backup now redirects you to the login page. |
+| PB-664 | PX-Backup now displays the status of a cluster as `Inactive`, even if the cluster has been deleted or is not reachable. |
+| PB-712 | Every time you update the cluster configuration, PX-Backup validates whether the cluster is accessible. |
+| PB-637 | If your license expires, PX-Backup pauses all scheduled backups until you apply a new license. |
+| PB-654 | PX-Backup now generates license files without prompting the user for the org name. <!-- Is this correct? --> |
```
Comment (bhavana-ar) [inline content/release-notes.md:48]: That is right, But it should be written as "license files generation no longer needs the user to provide the org name"

File: content/release-notes.md [hunk 2]
```diff
@@ -16,49 +16,49 @@ Sep 23, 2020
...
 | PB-671 | PX-Backup now displays the backup size for each namespace in the **Restore Backup** view. |
-| PB-478 | When a backup is in the delete or delete pending state, PX-Backup no longer displays the **View json** and **Show Details** options |
-| PB-302 | PX-Backup now automatically validates new backup locations when they are added. |
-| PB-634 | Users can now filter resources by resource type |
-| PB-747 | Administrators can now share backup locations with other users without disclosing the credentials or the bucket name by setting a backup location as the default backup location <!-- TODO: Revisit this -->|
+| PB-478 | When a backup is in the delete or delete pending state, PX-Backup no longer displays the **View json** and **Show Details** options. |
+| PB-302 | PX-Backup now automatically validates backup locations when the users add them. |
+| PB-634 | Users can now filter resources by resource type. |
+| PB-747 | Administrators can now share backup locations with other users without disclosing the credentials or the bucket name by setting a backup location as the default backup location. <!-- TODO: Revisit this -->|
```
Comment (bhavana-ar) [inline content/release-notes.md:39]: Sharing of default cloud account and backup location between admin and users

File: content/release-notes.md [hunk 3]
```diff
@@ -16,49 +16,49 @@ Sep 23, 2020
...
 | PB-768 | Users can now delete a resource without being prompted for the name of that resource. |
+| PB-636 | Improved clarity around the `OrgID` field in the **Add License** view. |
+| PB-645 | In the **Edit Backup Schedule View**,  you can now use a navigation link to go to the **Schedule Policy** view. |
+| PB-710 | Added a tooltip showing whether a backup schedule is being deleted. |
+| PB-509 | When your Keycloak token expires, PX-Backup now redirects you to the login page. |
+| PB-664 | PX-Backup now displays the status of a cluster as `Inactive`, even if the cluster has been deleted or is not reachable. |
+| PB-712 | Every time you update the cluster configuration, PX-Backup validates whether the cluster is accessible. |
+| PB-637 | If your license expires, PX-Backup pauses all scheduled backups until you apply a new license. |
+| PB-654 | PX-Backup now generates license files without prompting the user for the org name. <!-- Is this correct? --> |
+| PB-775 | PX-Backup now validates bucket names. |
```
Comment (bhavana-ar) [inline content/release-notes.md:49]: This can be removed since it is similar to PB-783.

File: content/release-notes.md [hunk 4]
```diff
@@ -16,49 +16,49 @@ Sep 23, 2020
...
 | **Improvement Number** | **Improvement Description** |
 |----|----|
-| PB-783 | PX-Backup now validates the bucket name field in the **Add Backup Location** view |
-| PB-831 | PX-Backup now displays the resources in alphabetical order in the **Create Backup** view |
-| PB-762 | When a backup schedule is in the delete pending state, PX-Backup no longer displays the remove, suspend, or edit options |
-| PB-837 | When a backup location is in the delete pending state, PX-Backup no longer displays the remove option |
-| PB-640 | The **Backups** view now features a progress bar indicating the progress of your backup operations |
-| PB-682 | The **Backup Rules** page now includes a help message explaining pre and post backup rules |
+| PB-783 | PX-Backup now validates the bucket name field in the **Add Backup Location** view. |
+| PB-831 | PX-Backup now displays the resources in alphabetical order in the **Create Backup** view. |
```
Comment (bhavana-ar) [inline content/release-notes.md:27]: Move this one to the bottom of the list

File: content/release-notes.md (hunk 3 above)
Comment (bhavana-ar) [reply content/release-notes.md:49]: PB-745: Add container option in px backup rule to specify which container the rule has to be executed on

File: content/release-notes.md [hunk 5]
```diff
@@ -16,49 +16,49 @@ Sep 23, 2020
...
+| PB-608 | Middleware not able to establish a connection with backup because the grpc connection was not closed. <br/><br/> **User impact:** Because of this issue, PX backup was marked as offline. <br/><br/> **Resolution:** Users will no longer see PX-backup marked as `offline` due to the middleware not being able to establish a connection with PX-Backup. | <!-- We already release noted this JIRA in 1.0.2, but I don't really understand it... -->
+| PB-664 | If a backup schedule is associated with an inactive cluster, users can not remove the cluster from PX-Backup. </br></br> **User impact:** PX-Backup displays an error message saying that the user can not delete the cluster. </br></br> **Resolution:**  The users can now select the inactive cluster and perform all operations except triggering a new backup. |
+| PB-744 | Sometimes, the **All backups** view did not show any backups, even if the users created multiple backups. </br></br> **User impact:** They could not see their backups in the **All backups** views. <br/><br/> **Resolution:** PX-Backup now shows all backups in the **All backups** view. |
```
Comment (bhavana-ar) [inline content/release-notes.md:61]: Issue: When there were large number of backups and each backup containing large number of resources, No backups were displayed in All backups page.
Resolution: All backup page now displays backups on an setup with large number of backups and resources.

File: content/release-notes.md [hunk 6]
```diff
@@ -16,49 +16,49 @@ Sep 23, 2020
 
 ### New features
 
-* <!-- PB-470 --> Clusters added using the CLI or API are now displayed on the pxcentral UI as well
-* <!-- PB-657 --> Starting version 1.1.0, we will have separate add cluster workflow for PX-Backup and Lighthouse.
+* If you add a new cluster using the CLI or API, PX-Backup now displays your cluster in the UI.
+* Added a separate Lighthouse view. <!-- Shall we link out to the PX-Central page-->
```
[the remark this answers is not in the document]
Comment (andreipope) [reply content/release-notes.md:29]: @bhavana-ar Done.

File: content/release-notes.md (hunk 2 above)
Comment (andreipope) [reply content/release-notes.md]: @bhavana-ar Done.

File: content/release-notes.md (hunk 4 above)
Comment (andreipope) [reply content/release-notes.md]: @bhavana-ar Done.

File: content/release-notes.md (hunk 5 above)
Comment (andreipope) [reply content/release-notes.md]: @bhavana-ar Done.

File: content/release-notes.md (hunk 3 above)
Comment (andreipope) [reply content/release-notes.md]: @bhavana-ar Done.

**Review by bhavana-ar — APPROVED** [review]

**Pull request merged**